src/Lecture3.hs

```diff
@@ -249,4 +298,7 @@ Just [8,9,10]
 [8,20,3]
 
 -}
-apply = error "TODO"
+apply :: Functor f => a -> f (a -> b) -> f b
+apply arg = fmap fn
+  where
+    fn ff = ff arg
```

Comment:
Absolutely correct solution 🥇 

If you want, you can write it shorter using lambda functions:

```suggestion
apply arg = fmap (\ff -> ff arg)
```